js/index.js

```diff
@@ -1 +1,79 @@
-// Your code goes here
+let navLinks = document.querySelectorAll("a");
+let headline = document.querySelector("h1");
+let subHeadings = document.querySelectorAll("h2");
+let contentSections = document.querySelectorAll(".content-section");
+let destination = document.querySelector(".content-destination");
+let image = document.querySelectorAll("img");
+
+headline.addEventListener("mouseenter", function(event) {
+  event.target.parentNode.style.color = "purple";
+});
+
+window.addEventListener("scroll", function(event) {
+  navLinks.forEach(x => (x.style.color = `rgba(${Math.floor(Math.random() * 225)}, ${Math.floor(Math.random() * 225)}, ${Math.floor(Math.random() * 225)}, 1)`));
+});
+
+navLinks[0].addEventListener("click", function(event) {
+  event.target.style["font-weight"] = 'bold';
+  event.preventDefault();
+});
+
+subHeadings[0].addEventListener("dblclick", function(event) {
+  event.target.style["font-style"] = "italic";
+});
+
+subHeadings[1].addEventListener("click", function(event) {
+  event.target.style["font-weight"] = "bold";
+  event.target.stopPropagation();
+});
+
+contentSections[0].querySelector(".text-content").querySelector("p").addEventListener("click", function(event) {
+  event.target.style["font-weight"] = "bold";
+  event.target.stopPropagation();
+});
+
+contentSections[0].querySelector(".text-content").querySelectorAll("p")[1].addEventListener("click", function(event) {
+  event.target.style["font-weight"] = "bold";
+  event.target.stopPropagation();
+});
+
+contentSections[1].querySelector(".text-content").addEventListener("mousemove", function(event) {
+  event.target.style["background-color"] = `rgba(${Math.floor(Math.random() * 225)}, ${Math.floor(Math.random() * 225)}, ${Math.floor(Math.random() * 225)}, 0.25)`;
```

Comment:
You repeat the same thing three times here; maybe make a function? For example:
```javascript
const randRGBval = () => Math.floor(Math.random() * 225);
contentSections[1].querySelector(".text-content").addEventListener("mousemove", (event) => {
  event.target.style["background-color"] = `rgba(${randRGBval()}, ${randRGBval()}, ${randRGBval()}, 0.25)`;
} 
```
or even:
```javascript
const randRGBval = () => Math.floor(Math.random() * 225);
const randRGB = (opacity = 1) => `rgba(${randRGBval()}, ${randRGBval()}, ${randRGBval()}, ${opacity})`;
contentSections[1].querySelector(".text-content").addEventListener("mousemove", (event) => {
  event.target.style["background-color"] = randRGB(0.25);
} 
```